saleor/checkout/urls.py

```diff
@@ -5,21 +5,18 @@
 
 checkout_urlpatterns = [
     url(r'^$', views.checkout_index, name='index'),
+    url(r'^details/$', views.checkout_details, name='details'),
+    url(r'^update/(?P<variant_id>\d+)/$',
+        views.update_checkout_line, name='update-line'),
+    url(r'^clear/$', views.clear_checkout, name='clear'),
+    url(r'^shipping-options/$', views.checkout_shipping_options,
+        name='shipping-options'),
     url(r'^shipping-address/', views.checkout_shipping_address,
         name='shipping-address'),
     url(r'^shipping-method/', views.checkout_shipping_method,
         name='shipping-method'),
-    url(r'^summary/', views.checkout_summary, name='summary'),
-    url(r'^remove_voucher/', remove_voucher_view,
+    url(r'^order-summary/', views.checkout_order_summary, name='order-summary'),
+    url(r'^summary/$', views.cart_summary, name='summary'),
```

Comment:
What this view does is rendering a "cart dropdown" HTML snippet, called via AJAX in one of storefront's JS files and injected to regular template. Let's change it to:

```suggestion
    url(r'^dropdown/$', views.checkout_dropdown, name='dropdown'),
```

BTW what an ugly pattern when looking at it from a perspective. I'm so happy that we're migrating to GraphQL/SPA and the old strorefront will be deprecated at some point.